src/components/progress/QueueCard.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { isFinite } from 'lodash-es';
+import { FormattedMessage } from 'react-intl';
+
+import Paper from '@material-ui/core/Paper';
+
+import Text from '../Text';
+
+function emphasizedH4(chunk) {
```

Comment:
Lol @ "chunk".

Reply:
even though it's not used normally in the JSX, this is still a component right? components should be Capitalized.

Reply:
might also consider moving this kind of stuff to its own file. do we need to have some react-intl utility components?

Reply:
indeed, `emphasized` in the other file is extremely similar

Reply:
> even though it's not used normally in the JSX, this is still a component right? components should be Capitalized.

It returns a functional component, but I don't believe that it is itself a component because it does not accept props.